docs/terminal/nowra.md

```diff
@@ -7,4 +7,50 @@
 Reserved.
 
 !!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Positions
+
+| Name               | ID      | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | --------------| -------------- | ---------------- | --------------------------------------|
+| Nowra TMA    | | Nowra Approach | 123.500          |NW_APP        |
+
+## Airspace
+
+### TMA Ownership
+
+The Terminal Control Area (TMA) owns the airspace within R421 up to `F125`; optionally R420F extends from `F125` to `F300`.
+This airspace is for facilitating military aircraft and low-level helicopter operations.
+
+### Activation of Restricted Areas
+
+When a controller logs in as Nowra TMA for R421:
+
+1. **Announce Activation**: Announce that R421 and other applicable PRDs are active.
+2. **Activate Surrounding Areas**: Notify surround controllers to activate the restricted areas to get the outlines visible.
+3. **Issue Clearances**: Treat restricted areas like R421 as Class C/D airspace and issue clearances accordingly.
+4. **Service Level**: Class C while R421 is active.
+
+To activate R420:
+1. Get an airspace release from BIK. Voiceless coord of all aircraft on published air routes via the NWA TACAN above `F125`.
```

Comment:
May need to have a look how this works with the WOL-LEECE track, but good enough for now. Just a note to revisit later. With the offshore PRDs active, tracking east of the TACAN is unavailable.

Reply:
I'll make a issue for this one to keep track